changelog.md

```diff
@@ -1,3 +1,10 @@
+2.8.0
+==========
+### New features:
+- ID Web works with Authority in place of Tenant ID and Domain. See [#2160](https://github.com/AzureAD/microsoft-identity-web/pull/2160)
+### Bug fixes:
+- Fixed a bug causing ClaimsIdentity.RoleClaimType to always be "roles" when using App Service Authentication. See [#2166](https://github.com/AzureAD/microsoft-identity-web/pull/2166)
+
```

Comment:
add the new version update for abstractions and add the Id Web supports CIAM authorities.

Reply:
done